ui-speedspacechart/src/components/helpers/drawElements/step.ts

```diff
@@ -4,9 +4,36 @@ import { clearCanvas, maxPositionValues } from '../../utils';
 
 const { CURVE_MARGIN_SIDES } = MARGINS;
 
+const STEP_HEIGHT = 6;
+const STEP_WIDTH = 1;
+const STEP_RADIUS = 0.3;
+
 export const drawStep = ({ ctx, width, height, store }: DrawFunctionParams) => {
   const { stops, ratioX, leftOffset } = store;
 
+  if (stops.length === 0) {
+    return;
+  }
+
+  let stopPosition =
```

Comment:
You can export `getStopPosition` from gridX.ts (or add it to utils) and use it here

Reply:
done